specs/unified_kernel_image.md

```diff
@@ -111,3 +111,9 @@ individual resources listed above at once, and their combination. Standard Linux
 [`sbsigntool`](https://manpages.debian.org/unstable/sbsigntool/sbsign.1.en.html) and
 [`pesign`](https://github.com/rhboot/pesign) can be used to sign UKI files. The signature format and process
 again match the ones already used for PE files, so they will not be redefined here.
+
+## Distribution-built UKIs Installed by Package Managers
+
+UKIs that are built centrally by distributions and installed via the package manager should be installed
+`/usr/lib/kernel/ukis/` with a filename `<version format specification>.efi`, so that tools staging or
```

Comment:
I wonder if we should allow multiarch too? Given we don't use the EFI arch suffix for now, they would clash otherwise

Reply:
I don't see a reason why not. Happy to change this, if you have a formulation. The filename scheme was just the first thing I thought of on the spot, since I assumed something like the name `6.1.0-7-amd64` I see on a Debian system in `/usr/lib/modules` could be used to accommodate this within the version format specification.

Reply:
yeah we can use the kernel package name which usually has the arch